setup.py

```diff
@@ -17,15 +22,15 @@ def test_reqs():
     author='Cera Davies',
     author_email='cera.davies@learnosity.com',
```

Comment:
We should probably update these Author details. Not sure what email we should use, though. Might need to double check with @sebablanco?

```python
setuptools.setup(
    author='Learnosity',
    author_email='???@learnosity.com',
    ...
)
```

Reply:
Lol yes, I noticed this and had made a note to find out what the email should be.

Reply:
@alangarf has just created `sdk@learnosity.com` for this.

Reply:
Cheers 😁 👍